### PR TITLE
Update readme [WIP]

### PR DESCRIPTION
**Pull request by dmey**
Improve current readme and add GPL text to in binary releases.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,56 @@
...
+Download and unzip the latest version of Avatar Animator from the [release page](https://github.com/letmaik/avatar-animator).
+
+## Getting started
+
+From the Avatar Animator folder, launch `avatar-animator.exe` before your preferred video-conferencing apps and select `OBS-Camera` as your input stream in your video-conferencing application settings. Examples on how to do this for common video-conferencing apps are given in the table below:
 
-### Prerequisites
+| Software Name   | Procedure                                                                                                                                     | Screenshot |
+| --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- |
+| Zoom            | Go to `Settings` -> `Video` and select `OBS-Camera` from the `Camera` drop-down menu.                                                         | TODO       |
```
Comment (letmaik) [inline README.md:47]: Have you checked that all of them are working?

File: LICENSE_BIN [hunk 2]
```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (letmaik) [inline LICENSE_BIN:1]: Did you check that this file becomes part of the package when running npm run package?

File: README.md [hunk 3]
```diff
@@ -1,15 +1,56 @@
...
+| Slack           | Start a call, then click on the gear icon in the top left corner of the call window and select `OBS-Camera` from the `Camera` drop-down menu. | TODO       |
 
-VirtualCam Avatar uses [obs-virtual-cam](https://github.com/Fenrirthviti/obs-virtual-cam/releases) which has to be installed separately. Note that the obs-virtual-cam installer assumes an OBS Studio installation and will fail otherwise. You can also download and extract the obs-virtual-cam zip package directly without installing OBS Studio. After unzipping, simply run `regsvr32 /n /i:1 "obs-virtualcam\bin\32bit\obs-virtualsource.dll"` from an elevated command prompt to install the virtual camera device. Use `regsvr32 /u "obs-virtualcam\bin\32bit\obs-virtualsource.dll"` to uninstall it again.
 
-### Download
+## Copyright and license
 
-You can download the latest release of VirtualCam Avatar from the GitHub releases.
+Copyright notice is stated at the top of source files. Source code released under [Apache-2.0](LICENSE). Binary releases released under [GPL-2.0](LICENSE_BIN).
```
Comment (letmaik) [inline README.md]: Could you add context on why GPL?

File: README.md (hunk 1 above)
Comment (dmey) [reply README.md:47]: no

File: LICENSE_BIN (hunk 2 above)
Comment (dmey) [reply LICENSE_BIN:1]: no

File: README.md (hunk 3 above)
Comment (dmey) [reply README.md]: Text revised

File: README.md (hunk 1 above)
Comment (letmaik) [reply README.md:47]: OK, well, at least for me Teams is not working. Zoom works, haven't checked the rest. Can you add an issue to verify all of them?

File: LICENSE_BIN (hunk 2 above)
Comment (letmaik) [reply LICENSE_BIN:1]: OK... Can you add an issue for that? I don't think it is automatic but the license should be part of the distributed package.

File: README.md [hunk 4]
```diff
@@ -1,15 +1,56 @@
-# VirtualCam Avatar
+<!-- omit in toc -->
+# Avatar Animator
```
Comment (letmaik) [inline README.md:2]: Could you also change the app title in the html file to the new name?

File: README.md (hunk 1 above)
Comment (dmey) [reply README.md:47]: https://github.com/letmaik/avatar-animator/issues/6

File: LICENSE_BIN (hunk 2 above)
Comment (dmey) [reply LICENSE_BIN:1]: see https://github.com/letmaik/avatar-animator/issues/7

File: README.md (hunk 4 above)
Comment (dmey) [reply README.md:2]: done in cda6cff

**Review by letmaik — APPROVED** [review]
Thanks!

**Pull request merged**